uc-crux-llvm/src/UCCrux/LLVM/Run/Simulate.hs

```diff
@@ -536,7 +536,17 @@ mkCallbacks appCtx modCtx funCtx halloc callbacks constraints cfg llvmOpts =
         IO r) ->
       IO r
     mkResultHook sym skipOverrideRef unsoundOverrideRef explRef memRef argShapeRef cruxResult resHook =
-      do unsoundness' <-
+      do for_ (traces cruxResult) $
+           \trace ->
+             do liftIO $ (appCtx ^. log) Hi "Trace:"
+                for_ trace $
+                  \loc ->
+                    let prettyLoc = ppProgramLoc loc
+                    in if prettyLoc == "internal"
```

Comment:
This relies on the implementation of `Position`'s `Pretty` output, right? Wouldn't it be less fragile to match against the `InternalPos` constructor instead?

Reply:
Yep, this is sheer laziness. Will fix.